web/src/app/dashboard/page.tsx

```diff
@@ -1312,9 +1310,9 @@ export default function Dashboard() {
                                     onClick={() => {
                                       shareOnWhatsApp(
                                         shareableLink,
-                                        "Hi! I'd love to get your feedback about our collaboration. You can share your testimonial here:"
+                                        "Hi! I'd love to get your feedback. You can share your testimonial here:"
```

Comment:
This phrase is being repeated in line 1330. Would it make sense to store it in a variable and reuse it?